README.md

```diff
@@ -7,7 +7,10 @@ This is a SMTP docker container for sending emails. You can also relay emails to
 The container accepts `RELAY_NETWORKS` environment variable which *MUST* start with `:` e.g `:192.168.0.0/24` or `:192.168.0.0/24:10.0.0.0/16`.
 The container accepts `KEY_PATH` and `CERTIFICATE_PATH` environment variable that if provided will enable TLS support. The paths must be to the key and certificate file on a exposed volume.  The keys will be copied into the container location.
 The container accepts `MAILNAME` environment variable which will set the outgoing mail hostname
-The container also accepts the `PORT` environment variable, to set the port the mail daemon will listen on inside the container. The default port is `25`.
+The container also accepts the `PORT` environment variable, to set the port the
+mail daemon will listen on inside the container. The default port is `25`.
```

Comment:
Looks like `mail daemon...` is on a newline, pls can u revert the newline added.